### PR TITLE
Improvements to info tracked in Wyre exceptions

### PR DESCRIPTION
**Pull request by jinchung**

### CODE REVIEW

File: src/handlers/wyre.js [hunk 1]
```diff
@@ -33,21 +33,33 @@ const wyreApi = axios.create({
...
 
-export const requestWyreApplePay = (
+export const getReferenceId = accountAddress => {
+  const addressLength = accountAddress.length;
+  const lowered = toLower(accountAddress);
+  return lowered.substring(addressLength - 12, addressLength);
+};
+
+export const showApplePayRequest = async (
+  referenceInfo,
```
Comment (brunobar79) [inline src/handlers/wyre.js:58]: should we keep the alphabetical order here?

File: src/handlers/wyre.js [hunk 2]
```diff
@@ -33,21 +33,33 @@ const wyreApi = axios.create({
...
+    super(`${referenceId}:${errorId}:${message}`);
+    this.name = name;
   }
 }
 
-export const requestWyreApplePay = (
+export const getReferenceId = accountAddress => {
+  const addressLength = accountAddress.length;
+  const lowered = toLower(accountAddress);
+  return lowered.substring(addressLength - 12, addressLength);
```
Comment (brunobar79) [inline src/handlers/wyre.js:54]: you can do return lowered.substr(-12) and avoid calculating the length of the string

Comment (jinchung) [reply src/handlers/wyre.js:54]: ah thank you! I was looking for something like that!

File: src/handlers/wyre.js (hunk 1 above)
Comment (jinchung) [reply src/handlers/wyre.js:57]: I'd like to keep the first argument to be the referenceInfo (as I am doing this for many functions) regardless of what the other params are named!

**Pull request merged**